open-metadata-implementation/server-chassis/server-chassis-spring/pom.xml

```diff
@@ -66,6 +66,30 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>open-lineage-janus-connector</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>search-integrator-api</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>elasticsearch-integration-connector</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
```

Comment:
you'll need to do the same in the build.gradle so that we pull these into the server chassis when built with gradle (may also apply to the api module above)